lib/Echidna/UI/Widgets.hs

```diff
@@ -57,22 +57,24 @@ campaignStatus (c@Campaign{_tests, _coverage, _ncallseqs}, uiState) = do
       hCenter underneath
     wrapInner inner =
       borderWithLabel (withAttr "bold" $ str title) $
-      summaryWidget _tests _coverage _ncallseqs
+      summaryWidget c
       <=>
       hBorderWithLabel (str "Tests")
       <=>
       inner
     title = "Echidna " ++ showVersion Paths_echidna.version
 
-summaryWidget :: [(SolTest, TestState)] -> CoverageMap -> Int -> Widget ()
-summaryWidget tests' coverage' ncallseqs' =
+summaryWidget :: Campaign -> Widget ()
+summaryWidget Campaign{_tests, _coverage, _ncallseqs, _genDict} =
```

Comment:
space before brace

can we maybe change this to use lens or use `NamedFieldPuns` to give these better names without underscores?